src/V3Tristate.cpp

```diff
@@ -55,6 +55,31 @@
 // duplicating vars and logic that is common between each instance of a
 // module.
 //
+//
+// Another thing done in this phase is signal strength handling.
+// Currently they are only supported in assignments and only in case when the strongest assignment
+// has constant with all bits equal on the RHS. It is the case when they can be statically
+// resolved.
+//
+// I do statical resolution in the following way:
+// - I find the assignment of value 0 (size may be greater than 1) that has greatest strength (the
+// one corresponding to 0) of all other assignments of 0.
+// - I do the same for value '1 and strength corresponding to value 1.
+// - I find greater of these two strengths. If they are equal I take the one that corresponds to 1.
+// - I remove all assignments that have strengths weaker or equal to the one I found. So I remove
+// all other assignments of constants on RHS and also all assignment that have both strengths
+// non-greater that the one I found, because they are weaker not matter what is on RHS.
```

Comment:
```suggestion
// Static resolution is done in the following way:
// - We find the assignment of value 0 (size may be greater than 1) that has greatest strength (the
// one corresponding to 0) of all other assignments of 0.
// - We do the same for value '1 and strength corresponding to value 1.
// - We find the greater of these two strengths. If they are equal we take the one that corresponds to 1.
// - We remove all assignments that have strengths weaker or equal to the one we found. So we remove
// all other assignments of constants on RHS and also all assignment that have both strengths
// non-greater that the one we found, because they are weaker no matter what is on RHS.
```
I think it's better to use _we_ here, or no pronouns at all. I think Wilson doesn't even like _we_, but I'm pretty sure it does appear in some comments in Verilator.